cdragontoolbox/__main__.py

```diff
@@ -441,6 +441,8 @@ def create_parser():
                                       help="dump a BIN file as a text tree")
     subparser.add_argument('-j', '--json', action='store_true',
                            help="extract to JSON")
+    subparser.add_argument('-V', '--version', default=1008, type=int,
+                           help="patch version this BIN file belongs to (default: %(default)s)")
```

Comment:
- The full option name could be `--patch-version` to be more explicit about what it should contain.
- It would be nice to accept `10.08` instead of `1008`. Internally we use an integer to ease comparison, but if the CLI argument is intended to receive a patch version (as it is documented), you should accept the doted version number and convert it afterwards.

Reply:
4fe04daf6aa4c0b9f8f1bdb1777b2d17e2f8b8b4 fine? that will allow both numbers like right now and normal version numbers with a single `.`, e.g. `9.2`.